UserInterface/UserInterface/UserInterface.cs

```diff
@@ -0,0 +1,156 @@
+        /// <param name="borderHeight">Border height.</param>
+        private void DrawWalls(int borderWidth, int borderHeight)
+        {
+            for (int currentColumn = 0; currentColumn < borderWidth; currentColumn++)
+            {
+                ShowMessage(ConstantsRepository.UpDownWall, false);
+            }
+
+            _window.SetCursorPosition(0, borderHeight);
+            
+            for (int currentColumn = 0; currentColumn < borderWidth; currentColumn++)
+            {
+                ShowMessage(ConstantsRepository.UpDownWall, false);
+            }
+
+            for (int currentRow = 0; currentRow < borderHeight; currentRow++)
+            {
+                _window.SetCursorPosition(0, currentRow);
+                ShowMessage(ConstantsRepository.LeftRightWall, false);
+            }
+
+            for (int currentRow = 0; currentRow < borderHeight; currentRow++)
+            {
+                _window.SetCursorPosition(borderWidth, currentRow);
+                ShowMessage(ConstantsRepository.LeftRightWall, false);
+            }
+        }
+
+        /// <summary>
+        /// Draws border's corners.
+        /// </summary>
+        /// <param name="borderWidth">Border width.</param>
+        /// <param name="borderHeight">Border height.</param>
+        private void DrawCorners(int borderWidth, int borderHeight)
+        {
+            _window.SetCursorPosition(0, 0);
+            ShowMessage(ConstantsRepository.UpLeftCorner, false);
+            
+            _window.SetCursorPosition(borderWidth, 0);
+            ShowMessage(ConstantsRepository.UpRightCorner, false);
+
+            _window.SetCursorPosition(0, borderHeight);
+            ShowMessage(ConstantsRepository.DownLeftCorner, false);
+
+            _window.SetCursorPosition(borderWidth, borderHeight);
+            ShowMessage(ConstantsRepository.DownRightCorner, false);
+        }
+
+        /// <summary>
+        /// Draws a game board.
+        /// </summary>
+        /// <param name="board">Game board.</param>
+        public void DrawBoard(Board board)
+        {
+            for (int currentRow = 0; currentRow < board.GameBoard.GetLength(1); currentRow++)
+            {
+                _window.SetCursorPosition(ConstantsRepository.OffsetX - 1, currentRow + ConstantsRepository.OffsetY);
+
+                for (int currentColumn = 0; currentColumn < board.GameBoard.GetLength(0); currentColumn++)
+                {
+                    Console.Write(board.GameBoard[currentColumn, currentRow]);
+                    _window.SetCursorPosition(ConstantsRepository.OffsetX + currentColumn, currentRow + ConstantsRepository.OffsetY);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets input key. 
+        /// </summary>
+        /// <returns>Pressed button value.</returns>
+        public ConsoleKey? GetInputKey()
+        {
+            if (Console.KeyAvailable)
+            {
```

Comment:
This can be one liner `return Console.KeyAvailable ? Console.ReadKey(true).Key : null`

Reply:
It will be fixed in the next PR.